checker/src/main/java/org/checkerframework/checker/initialization/InitializationAnnotatedTypeFactory.java

```diff
@@ -518,7 +522,7 @@ protected AnnotationMirror getFreeOrRawAnnotationOfSuperType(TypeMirror type) {
                 annotation = createUnclassifiedAnnotation(superClass);
             }
         } else {
-            // Use Object as a valid super-class
+            // Use Object as a valid super-class, using overloaded version of constructors
```

Comment:
It's not quite clear what the `constructors` refers to here, as it calls two methods below. Maybe clarify?